5/Core/LearningJava.iml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please remove this file from this folder, it's not necessary

Reply:
ok